<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ap</class>
 <widget class="QDialog" name="ap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="PB_anadir">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>630</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>AÑADIR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_Titulo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INSERCIÓN DE PELICULAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-230</y>
     <width>1861</width>
     <height>1211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imag/Downloads/foto2 (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TITULO</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="accion">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ACCIÓN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>160</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GENERO</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="aventura">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AVENTURA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="comedia">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>530</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>COMEDIA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drama">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>300</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DRAMA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ciencia">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>420</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CIENCIA FICCIÓN</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fantasia">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>530</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FATASIA</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="thriller">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>300</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>THRILLER</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="horror">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>420</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>HORROR</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="misterio">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>530</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MISTERIO</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AÑO</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1500</number>
   </property>
   <property name="maximum">
    <number>2025</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>355</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DIRECTOR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="QLE_Director">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>425</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RATING</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>-170</x>
     <y>-60</y>
     <width>1581</width>
     <height>981</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/foto2 (1).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>71</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/accion.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>340</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/aventura.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/comedia.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/drama.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>350</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/ciencia_ficcion.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>460</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/fantasia.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>230</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/thriller.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>350</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/horror.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>460</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/imagenes/misterio.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>label_2</zorder>
  <zorder>PB_anadir</zorder>
  <zorder>QLE_Titulo</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>accion</zorder>
  <zorder>label_4</zorder>
  <zorder>aventura</zorder>
  <zorder>comedia</zorder>
  <zorder>drama</zorder>
  <zorder>ciencia</zorder>
  <zorder>fantasia</zorder>
  <zorder>thriller</zorder>
  <zorder>horror</zorder>
  <zorder>misterio</zorder>
  <zorder>label_5</zorder>
  <zorder>spinBox</zorder>
  <zorder>label_6</zorder>
  <zorder>QLE_Director</zorder>
  <zorder>label_7</zorder>
  <zorder>doubleSpinBox</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
